### PR TITLE
CUNO-1853 / ADF-695: User without access to assets can save images trough the item

### PR DESCRIPTION
**Pull request by clagosarias**
**Jira issue**
https://oat-sa.atlassian.net/browse/ADF-695

**Related PR**
https://github.com/oat-sa/tao-core-ui-fe/pull/368

**AC**
- Preselect item gallery when user doesn't have access to Assets.
- Preselect Assets when user does have access.
- Make "Add file" button appear without having to click a folder. (If permissions allow it).

**How to test**
- Make access to Assets forbidden for user. (By restricting access only to Global Manager role) (make it recursive)
- Login with user and author an item
- Try to insert media/image

### CODE REVIEW

**Comment by AntonNeviarouski** [comment]
QA verified

File: views/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         "@oat-sa/tao-core-libs": "0.4.3",
         "@oat-sa/tao-core-sdk": "1.15.0",
         "@oat-sa/tao-core-shared-libs": "1.0.3",
-        "@oat-sa/tao-core-ui": "1.36.0",
+        "@oat-sa/tao-core-ui": "git://github.com/oat-sa/tao-core-ui-fe#fix/ADF-695/item-gallery-default",
```
Comment (habemuscode) [inline views/package.json:20]: Don't forget to change

**Review by habemuscode — APPROVED** [review]
LGTM

**Review by 404passionFruit — APPROVED** [review]

File: views/package-lock.json [hunk 2]
```diff
@@ -772,9 +772,8 @@
             "integrity": "sha512-S6/2ol3NpT99su9ZlZgoVvZHKW+yRVro562Ab1pizyCwv2YfO1r2FbpEDTS/cgfqI2JvgMtuP1V1SjR3T8Avrg=="
         },
         "@oat-sa/tao-core-ui": {
-            "version": "1.36.0",
-            "resolved": "https://registry.npmjs.org/@oat-sa/tao-core-ui/-/tao-core-ui-1.36.0.tgz",
-            "integrity": "sha512-4yzT8vy5AsSwJqAg7wPELQ1D2rtJ6BOPlCN9ilF4XoBoICa9ZCpGlILWGacfAyjzTFuOddJSgRdUY2uv8XvC9w=="
+            "version": "git://github.com/oat-sa/tao-core-ui-fe.git#1deeddfcb3b5b7733a0067b45a333f7204b12637",
```
Comment (404passionFruit) [inline views/package-lock.json:775]: Please update this one :)

**Pull request merged**